.gitignore

```diff
@@ -25,7 +25,7 @@ cachegrind.out*
 # Codelite (among others) project files
 *.project
 # ctags index files
-tags
+*tag*
```

Comment:
I think this glob is not ideal and can lead to confusion, since it includes *any* file that includes the substring `tag` in its name, e.g. `sys/include/staggeringly_long_header_name.h`